Dockerfile

```diff
@@ -79,3 +61,9 @@ RUN pip3 install \
   beautifulsoup4==4.7.1 \
   lxml==4.3.1 \
   pycodestyle==2.5.0
+
+# Install ruby deps with bundler to make things more standard for Ruby folks.
+RUN gem install bundler:2.0.1
+RUN bundle config --global silence_root_warning 1
+COPY Gemfile* /
```

Comment:
Is there a reason you dockerignore the Gemfile and then copy it explicitly?

Reply:
I dockerignore everything *but* the Gemfile. There are tons of files in the repo and they don't need to go to docker, but it needs the Gemfile and the lock.

Reply:
oops, sorry missed the `!` in the `dockerignore`.